--- old/Term/ReadLine/Gnu/XS.pm	2005-02-15 03:00:35.000000000 +0100
+++ new/Term/ReadLine/Gnu/XS.pm	2005-02-15 03:00:43.000000000 +0100
@@ -548,7 +548,7 @@
     $_i = $state ? $_i + 1 : 0;	# clear counter at the first call
     my $cw = $Attribs{completion_word};
     for (; $_i <= $#{$cw}; $_i++) {
-	return $cw->[$_i] if ($cw->[$_i] =~ /^\Q$text/);
+	return $cw->[$_i] if ($cw->[$_i] =~ /^\Q$text/i);
     }
     return undef;
 }
@@ -577,7 +577,7 @@
     }
 
     for (; $_i <= $#_matches; $_i++) {
-	return $_matches[$_i] if ($_matches[$_i] =~ /^\Q$text/);
+	return $_matches[$_i] if ($_matches[$_i] =~ /^\Q$text/i);
     }
     return undef;
 }
